pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -207,26 +110,52 @@ def map_fips(self, fips: str) -> None:
             return
         population = self._get_population(fips)
 
-        # Get multiplicative conversion factors to scale model output to fit dataset current values
-        hosp_rescaling_factor, icu_rescaling_factor = self._get_model_to_dataset_conversion_factors(
-            t0_simulation=t0_simulation, fips=fips, pyseir_outputs=pyseir_outputs,
-        )
-
         # We will shim all suppression policies by the same amount (since historical tracking error
         # for all policies is the same).
         baseline_policy = "suppression_policy__inferred"  # This could be any valid policy
-        model_death_ts = pyseir_outputs[baseline_policy]["total_deaths"]["ci_50"]
+
         # We need the index in the model's temporal frame.
         idx_offset = int(fit_results["t_today"] - fit_results["t0"])
-        # Get the latest observed values to shim to.
+        # Get the latest observed values to use in calculating shims
         observed_latest_dict = combined_datasets.get_us_latest_for_fips(fips)
         shim_log = structlog.getLogger(fips=fips)
 
-        death_shim = shim.shim_deaths_model_to_observations(
-            model_death_ts=model_death_ts,
-            idx=idx_offset,
-            observed_latest=observed_latest_dict,
-            log=shim_log,
+        # For Deaths
```

Comment:
Do you think it would be helpful at all to pull this code block here out into a function that's maybe like?
``` death_shim, total_host_shim, icu_shim = calculate_shims(...)```

I think it would give us a chance to maybe make the shim calculation more testable + easier to rip out if we want

Reply:
Fair point, but I think I'll leave it as for now. I'd like to continue to treat the three shims independently for now, since they serve three different parts of the front end. But testing them as a group on the same data object makes sense.